Maco Fixes

tgentet-obrien opened the pull request:
Fixed build errors with MACRO and Minimize image errors

tgentet-obrien committed:
Maco Fixes

dannye commented:
btw, the build errors result from using the wrong rgbds version. This repo is compatible with rgbds v0.4.2 (the version specified in INSTALL.md).
In order to make this compatible with newer versions of rgbds, the best thing to do is pull from [pret/pokeyellow](https://github.com/pret/pokeyellow) instead of manually trying to recreate all the changes.
I'll get around to doing this. So thanks for the nudge.

tgentet-obrien commented:
@dannye the changes I added into the PR allow it to work on the latest build. I've already been messing with it and added a new Pokemon in, welcome [Karen](https://github.com/tgentet-obrien/pokeyellow-gen-II/tree/Karen) 👀
